docs/production-deployment/cloud/service-health.mdx

```diff
@@ -0,0 +1,113 @@
+---
+id: service-health
+title: Service health
+sidebar_label: Service health
+description: Use Temporal Cloud metrics to monitor your production deployment Service health.
+toc_max_heading_level: 4
+tags:
+  - temporal-cloud
+  - metrics
+  - observability
+  - health
+---
+
+Temporal Cloud metrics help you to monitor the Service health of your production deployments.
+
+## Monitor availability issues
```

Comment:
seems like a remediation step for this "how to" is enabling Multi-region Namespaces?

Reply:
It's an excellent remediation step.